src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,39 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
 import { Loader } from '../Loader';
+import { TodoModalProps } from './TodoModal.types';
+import { getUser } from '../../utils/getUser';
+
+export const TodoModal: React.FC<TodoModalProps> = ({
+  selectedTodo,
+  setSelectedTodo,
+}) => {
+  const [userName, setUserName] = useState('');
+  const [userIsLoading, setUserIsLoading] = useState(false);
```

Comment:
a bit strange that the test with the loader fails w/o timeout
try to set it true by default

Reply:
Doesn't work